modules/300-prometheus/hooks/generate_password.go

```diff
@@ -20,4 +20,11 @@ import (
 	"github.com/deckhouse/deckhouse/go_lib/hooks/generate_password"
 )
 
-var _ = generate_password.RegisterHook("prometheus")
+const (
+	moduleValuesKey = "prometheus"
+	authSecretNS    = "d8-monitoring"
+	authSecretName  = "basic-auth"
+)
+
+var hook = generate_password.NewBasicAuthPlainHook(moduleValuesKey, authSecretNS, authSecretName)
```

Comment:
This is a global value declaration. Can we avoid it?

Reply:
Done